Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -389,16 +398,34 @@ extension ContentBlockerHelper {
 extension ContentBlockerHelper {
 
     static func setTrackingProtectionMode(_ enabled: Bool, for prefs: Prefs, with tabManager: TabManager) {
-        guard let isPrivate = tabManager.selectedTab?.isPrivate else { return }
-        let key = isPrivate ? ContentBlockingConfig.Prefs.PrivateBrowsingEnabledKey : ContentBlockingConfig.Prefs.NormalBrowsingEnabledKey
+        guard let selectedTab = tabManager.selectedTab else {
+            return
+        }
+
+        let key: String
+
+        switch selectedTab.type {
+        case .regular:
+            key = ContentBlockingConfig.Prefs.NormalBrowsingEnabledKey
+        case .private:
+            key = ContentBlockingConfig.Prefs.PrivateBrowsingEnabledKey
+        }
+
         prefs.setBool(enabled, forKey: key)
```

Comment:
Would agree should be a new ticket if switching to prefs

Reply:
Ok, we'll make a new ticket. Probably would be easiest to just remove `Prefs` all together and see how big of a task it would be to migrate all over to `Preferences`.

I don't expect there'll be that many preferences left (that are not FF related and have to be kept)

Reply:
👍